src/routes/v2/authenticatedSites/settings.js

```diff
@@ -136,7 +135,7 @@ class SettingsRouter {
     router.post(
       "/repo-password",
       this.authorizationMiddleware.verifyIsEmailUser,
-      attachWriteRouteHandlerWrapper(this.updateRepoPassword)
+      attachRollbackRouteHandlerWrapper(this.updateRepoPassword)
     )
```

Comment:
@alexanderleegs may I confirm that read is fine here? this seems to be infra level stuff, and therefore does not warrent rollbackhandler/write handler?